sdformat5.rb

```diff
@@ -0,0 +1,67 @@
+class Sdformat5 < Formula
+  desc "Simulation Description Format"
+  homepage "http://sdformat.org"
+  url "http://gazebosim.org/distributions/sdformat/releases/sdformat-5.0.0.tar.bz2"
+  sha256 "d3fb29392a0b9878eaaf7c9aec144f5ca65bba4649bda15bd5d4603d7bb58f10"
+
+  head "https://bitbucket.org/osrf/sdformat", :branch => "default", :using => :hg
+
+  bottle do
+    root_url "http://gazebosim.org/distributions/sdformat/releases"
+  end
```

Comment:
I think we need to add placeholders for each os version (see https://github.com/osrf/homebrew-simulation/pull/189/commits/41425b4d6e3be669eacdfbd38ea8dac6ba0add41#diff-cf6310d84e47d0268f732d6a34c364d6R12 )